pymbar/mbar.py

```diff
@@ -404,27 +405,25 @@ def computeEffectiveSampleNumber(self, verbose = False):
         return N_eff
 
     # =========================================================================
-    def computeOverlap(self, return_dict=True):
+    def computeOverlap(self):
         """
         Compute estimate of overlap matrix between the states.
 
         Parameters
         ----------
-        return_dict : bool, Default False
-            If true, results are a dict, else a tuple
+        None
```

Comment:
If there are not parameters, I think we can remove this section from the docstringl

Reply:
Eh, I think it helps to be explicit?